styles/theme.scss

```diff
@@ -24,6 +24,8 @@
     --network-color-stroke-dark: rgb(211, 48, 121);
     --network-color-transparent-light: rgb(211, 48, 121, 0.05);
```

Comment:
Please correct the `rgb` values with `rgba` where alpha is used. This error exists in your additions and in the existing transparent values.